yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -361,6 +361,23 @@ void checkResultInternal(@Nonnull final Object actual, @Nonnull final String que
         };
```

Comment:
mildly related: `getSupportedVersionConfig` can now be refactored to avoid anonymous class definition:  
```
    private static QueryConfig getSupportedVersionConfig(final Object value, final int lineNumber, final YamlExecutionContext executionContext) {
        final SupportedVersionCheck check = SupportedVersionCheck.parse(value, executionContext);
        if (!check.isSupported()) {
            return new SkipConfig(QUERY_CONFIG_SUPPORTED_VERSION, value, lineNumber, executionContext, check.getMessage());
        }
        return getNoOpConfig(lineNumber, executionContext);
   }
```

Reply:
Sure, will do in a subsequent PR.